sdks/python/pytest.ini

```
@@ -70,6 +70,7 @@ markers =
     uses_mock_api: tests that uses the mock API cluster.
     uses_feast: tests that uses feast in some way
     gemini_postcommit: gemini postcommits that need additional deps.
+    require_non_self_hosted: tests requiring specialized environments or infrastructure not present in self-hosted setups.
```

```suggestion
    require_non_self_hosted: tests which can't be run on Beam's self hosted GitHub Actions runners. Usually this is because of issues with docker-in-docker or other environmental challenges. Context: https://github.com/apache/beam/pull/35585
```

It would help to have more info here, feel free to use different wording though.